integreat_cms/cms/templates/pois/poi_form_sidebar/opening_hours_box.html

```diff
@@ -0,0 +1,28 @@
+{% extends "../../_collapsible_box.html" %}
+{% load i18n %}
+{% load static %}
+{% load content_filters %}
+{% load widget_tweaks %}
+{% block collapsible_box_icon %}
+    calendar-clock
+{% endblock collapsible_box_icon %}
+{% block collapsible_box_title %}
+    {{ poi_form.opening_hours.label }}
+    <br />
+    ({% translate "Currently not visible in apps" %})
+{% endblock collapsible_box_title %}
+{% block collapsible_box_content %}
+    <opening-hours-widget></opening-hours-widget>
+    {{ opening_hour_config_data|json_script:"openingHourConfigData" }}
+    {{ poi_form.opening_hours.value|json_script:"openingHourInitialData" }}
+    {# This hidden input is required for the change detection of the opening hours field #}
+    <input type="hidden"
+           id="initial-id_opening_hours"
+           name="initial-opening_hours"
+           value="{{ poi_form.opening_hours.value }}"/>
+    <div class="py-2 px-4">
+        {% render_field poi_form.temporarily_closed %}
+        <label for="{{ poi_form.temporarily_closed.id_for_label }}">{{ poi_form.temporarily_closed.label }}</label>
+        <div class="help-text">{{ poi_form.temporarily_closed.help_text }}</div>
+    </div>
+{% endblock collapsible_box_content %}
```

Comment:
The opening hours box has too much padding now:

| Before | After |
| --- | --- |
| ![Screenshot 2023-01-13 at 20-59-51 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/212408963-d7ef8fef-c45a-4322-a6fc-be891bce6a34.png) | ![Screenshot 2023-01-13 at 20-58-58 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/212408984-d25b90bd-a1d9-4647-90be-b2ecf070e10e.png) |

(Notice the white gap between the day rows)
Probably you have to add another variable to the collapsible box content to allow to use it without the `p-4` padding for the entire content div...

Reply:
Oh yes, thank you so much for this hind. I created a new variable that allows to have the box content without the extra padding